### DISCUSSION TITLE
Skin generated damaged or incomplete.

### DISCUSSION THREAD

**Discussion by agrecascino** [The-Wineskin-Project/WineskinServer#360] [Q&A]
Hi, 

I'm running Mojave on my hackintosh. I have gatekeeper disabled, though I've also removed the quarantine flags on the folders. I'm on a case-sensitive filesystem as well. Whenever I generate wrappers, they're damaged or incomplete, and running wineskin inside of them just hangs when I attempt to install something. Older versions of wineskin seem to work a little better, but still just hang when you actually try to install anything(though I'm pretty sure I've made it thru an installation on porting kit once?). I'm basically totally lost now.

**Comment by Gcenx** [comment]
The default macOS file system is case insensitive that might be why your having issues.
